unit/size_left_test.c

```diff
@@ -301,13 +301,13 @@ int main(int argc, char **argv)
 	if (!hints)
 		return EXIT_FAILURE;
 
-	while ((op = getopt(argc, argv, "f:a:h")) != -1) {
+	while ((op = getopt(argc, argv, "f:p:h")) != -1) {
 		switch (op) {
-		case 'a':
+		case 'f':
 			free(hints->fabric_attr->name);
 			hints->fabric_attr->name = strdup(optarg);
 			break;
-		case 'f':
+		case 'p':
 			free(hints->fabric_attr->prov_name);
 			hints->fabric_attr->prov_name = strdup(optarg);
 			break;
```

Comment:
The above parsing code which is present in all the unit tests can be replaced by calling ft_parseinfo and adding parsing fabric name to the function.